### PR TITLE
Makes removePersonFromEvent remove completely and fixes bug with index validation checking

### LINKED ISSUE #301: Remove person from event only removes from 1 role if person has multiple roles

### LINKED ISSUE #303: Remove from Event wrong index checking logic
image
image

### PR DESCRIPTION
**Pull request by jan-kai1**
Closes #303 , Closes #301 

Changes the index validation in RemoveFromEvent and removes person from all roles instead of just 1

### CODE REVIEW

**Review by cshao02 — APPROVED** [review]
LGTM!

**Review by shardhrv — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/logic/commands/event/commands/RemovePersonFromEventCommandTest.java [hunk 1]
```diff
@@ -194,6 +196,26 @@ public void execute_lastPredicateRelated() {
...
+
+        eventManager.getEventList().get(0).addPerson(testMan, "vendor");
+        eventManager.getEventList().get(0).addPerson(testMan, "attendee");
+        model.addPerson(testMan);
+        assertEquals(eventManager.getEventList().get(0).getAllPersons().size(), 7);
+        RemovePersonFromEventCommand removePersonFromEventCommand2 = new RemovePersonFromEventCommand(
+                Index.fromOneBased(1),
+                Index.fromZeroBased(model.getFilteredPersonList().size() - 1));
+        removePersonFromEventCommand2.execute(model, eventManager);
+        //check no. of people in TECH_CONFERENCE
```
Comment (brianchoon) [inline src/test/java/seedu/address/logic/commands/event/commands/RemovePersonFromEventCommandTest.java:214]: Informative comment so maintainers can understand the code!

**Review by brianchoon — APPROVED** [review]
LGTM

**Pull request merged**